hugo.yaml

```diff
@@ -89,25 +90,25 @@ params:
         improve</a>.
   links: # CUSTOMIZE
     user:
-      - name: GitHub Discussions
-        url: https://github.com/google/docsy/discussions
+      - name: GitHub community
+        url: https://github.com/theupdateframework/community
         icon: fa-brands fa-github
         desc: Discussion and help from your fellow users
```

Comment:
This entry doesn't really make sense since there are no discussions enabled over the community repo.

Reply:
Okay. What should GitHub repo would be best here?

Reply:
Drop this entry. Readers can sign up on Slack and/or the Google group.